eng/common/templates/jobs/source-build.yml

```diff
@@ -14,7 +14,7 @@ parameters:
   # This is the default platform provided by Arcade, intended for use by a managed-only repo.
   defaultManagedPlatform:
     name: 'Managed'
-    container: 'mcr.microsoft.com/dotnet-buildtools/prereqs:centos-7-3e800f1-20190501005343'
+    container: 'mcr.microsoft.com/dotnet-buildtools/prereqs:centos-stream8'
```

Comment:
These changes are in eng/common

Reply:
I'm lacking context here. Are you suggesting I revert this? Or what action should I take?

Reply:
```suggestion
    container: 'mcr.microsoft.com/dotnet-buildtools/prereqs:centos-7-3e800f1-20190501005343'
```

Reply:
Juan is saying that eng/common/ changes must be done in arcade instead of in the underlying repositories: https://github.com/dotnet/arcade/blob/05a8ec6db75694246db7a7b9ab680c9be91bf1d6/eng/common/templates/jobs/source-build.yml#L17